google-cloud/lib/google/cloud/version.rb

```diff
@@ -15,6 +15,6 @@
 
 module Google
   module Cloud
-    VERSION = "0.58.0".freeze
+    VERSION = "0.2.0".freeze
```

Comment:
Needs to be `0.59.0`.

Reply:
Fixed manually. This appears to be a bug in releasetool.